lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -809,6 +809,16 @@ module Resources
         'op': 'replace',
         'path': '/name/0/family',
         'value': 'Potter'
+      },
+      {
```

Comment:
how is there another test operation with a different id?

Reply:
removed duplicate https://github.com/cerner/fhir.cerner.com/commit/066c7292a003ece97caf430530f337d0a926256f

Reply:
screenshot here https://github.com/cerner/fhir.cerner.com/pull/539#issuecomment-790772050